.buildkite/data.rayci.yml

```diff
@@ -66,10 +66,31 @@ steps:
     depends_on: datanbuild
     job_env: forge
 
-  - label: ":database: data: doc gpu"
+  - label: ":database: data: doc tests"
     tags: 
       - data
-      - ci_doc
+      - doc
+    instance_type: medium
+    commands:
+      # doc tests
+      - bazel run //ci/ray_ci:test_in_docker -- python/ray/... //doc/... data 
+        --build-name data12build
+        --except-tags gpu
```

Comment:
Where do we run the GPU doctests?

Reply:
there is the `data: doc gpu tests` down below. They use a different machine type so need to be in a different job.

Reply:
Ah, okay. Some follow up questions:
* How come we have two `bazel run` for CPU but only one for GPU?
* Do we have GPU doctests for Train and the other libraries?
* Are GPU tests run for `python/ray/...`? Train has a few GPU docstring examples https://github.com/ray-project/ray/blob/79a3c58f5cd0a81422f917fc2b22f7272bb98981/python/ray/train/BUILD#L19-L28

Reply:
All good questions!

> How come we have two bazel run for CPU but only one for GPU?
- All gpu doc tests and examples are under //doc/... so for gpu there is just one job. For cpu, some doctest are under //python/ray/data/... so i need to split them into 2 jobs

> Do we have GPU doctests for Train and the other libraries?
- serve: https://github.com/ray-project/ray/blob/master/.buildkite/serve.rayci.yml#L107
- train: https://github.com/ray-project/ray/blob/master/.buildkite/ml.rayci.yml#L51
- data: https://github.com/ray-project/ray/blob/master/.buildkite/data.rayci.yml#L69
- core and rllib doesn't have gpu doc tests

> Are GPU tests run for python/ray/...? Train has a few GPU docstring examples
- train is the only one that has gpu docstring in python/ray/... and is covered in https://github.com/ray-project/ray/blob/master/.buildkite/ml.rayci.yml#L51